spec/models/task_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Task, type: :model do
+  subject(:task) { Task.create(title: "Short text", description: "Long text", due_date: Date.today + 1.day ) }
```

Comment:
It can stay as it is, but imho breaking line before each key is more readable:

```ruby
subject(:task) do
  Task.create(title: "Short text",
              description: "Long text",
              due_date: Date.today + 1.day )
end
```